lib/file-finder.js

```diff
@@ -113,20 +114,23 @@ class FileFinder {
                         // Add the file path to the cache of each directory searched.
                         for (let j = 0; j < searched; j++) {
                             cache[dirs[j]].push(filePath);
+                            yield [filePath];
```

Comment:
yeah so the config is empty and it results npm run test failing
<img width="801" alt="screen shot 2017-05-27 at 10 40 56 pm" src="https://cloud.githubusercontent.com/assets/3211873/26525689/92e2da9c-432d-11e7-8528-d2cdc0e4e48a.png">

I think the screenshot happens when the rules has no values